clanlib-glibc2.34.patch

```diff
@@ -0,0 +1,56 @@
+From 393c23cce9f16d4d5097ae8196f2612f93ed99ee Mon Sep 17 00:00:00 2001
```

Comment:
shouldn't this patch go into the shared-module?

Reply:
Both should, but the shared-module is from @Grumbel and they are inactive.

Reply:
Issues are also blocked https://github.com/Grumbel/shared-modules/

Reply:
oh I didn't realize it was a different one... forget about it then.